config.js

```diff
@@ -7,19 +7,23 @@ const configVars = {
 const configInit = [
   {
     id: configVars.ADMIN_TOKEN,
-    value: 'admin'
+    value: 'admin',
+    display: 'Admin token'
```

Comment:
Can these `display` be made a little more descriptive? Admin token is used to allow incoming requests to use a fixed bearer token for testing the interactions, should be unset in production. Data trust service is the base endpoint of the data trust service. Require auth is whether or not authorization is required to access endpoints on the BSA. Require auth out is whether the BSA will fetch and use a token when making external requests. Feel free to rework the text or designs as needed for space reasons

Reply:
Perhaps a display and also a helper tooltip with a little extra text to explain what the value is/used for?